public/widgets/paper-badger-widget.js

```diff
@@ -87,22 +87,44 @@ function splitArray(arr, lookGroup, taxonomyClass)
 	return ttt;
 }
 
+function getEndpoint(confIn, count)
```

Comment:
Hi both!

I'm going to r+ and merge this since it works and it's a great feature.

However, I would prefer clearer semantics help onboard new contributors (clever semantics can make it hard for new ppl to join in :smile:)